src/rf/apps/workers/process.py

```diff
@@ -42,6 +43,7 @@
 ERROR_MESSAGE_JOB_TIMEOUT = 'Layer could not be processed. ' + \
                             'The job timed out after ' + \
                             str(math.ceil(TIMEOUT_SECONDS / 60)) + ' minutes.'
+ERROR_MESSAGE_EMR_DEAD = 'Image processing failed.'
```

Comment:
I don't think it really matters, it will be shown to the user so whatever makes it clear about where we went wrong without giving too much detail is fine. I'll change it simply "Processing failed."